share/spice/cryptocurrency/cryptocurrency.js

```diff
@@ -0,0 +1,250 @@
+(function (env) {
+    "use strict";
+    
+    // Currencies that we have flags for.
+    // These correspond to traditional currencies of countries.
+    var currency2country = {
+        "aud": true,
+        "cny": true,
+        "eur": true,
+        "gbp": true,
+        "hkd": true,
+        "jpy": true,
+        "nzd": true,
+        "pln": true,
+        "rur": true,
+        "sgd": true,
+        "usd": true
+    };
+    
+    // Cryptocurrencies that we have flags for.
+    var crypto_flags = {
+        "btc": true,
+        "doge": true,
+        "drk": true,
+        "ltc": true,
+        "xrp": true
+    }
```

Comment:
Currently looks like:

![screen shot 2015-04-08 at 4 33 26 am](https://cloud.githubusercontent.com/assets/81969/7041576/7b993daa-dda8-11e4-8425-1508f3630782.png)

But should be:

![screen shot 2015-04-08 at 4 33 44 am](https://cloud.githubusercontent.com/assets/81969/7041579/803f5ab0-dda8-11e4-94b6-83e067f0ec71.png)